apps/mail/lib/themes/theme-utils.ts

```
@@ -0,0 +1,510 @@
+      // Extract radius (usually the same as light mode)
+      const radiusMatch = darkContent.match(/--radius:\s*([^;]+);/);
+      if (radiusMatch) {
+        colors.radius = radiusMatch[1].trim();
+      }
+
+      // Extract panel color (fallback to background if not found)
+      const panelMatch = darkContent.match(/--panel:\s*([^;]+);/);
+      if (panelMatch) {
+        colors.background = hslToHex(panelMatch[1].trim());
+      } else {
+        colors.background = colors.background;
```

Biome (1.9.4)

[error] background is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)